<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>toResultListFormatUI</class>
 <widget class="QDialog" name="toResultListFormatUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>401</width>
    <height>293</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Choose format</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0">
    <widget class="QLabel" name="TextLabel1">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>&amp;Format of Data:</string>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
     <property name="buddy">
      <cstring>formatCombo</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="3">
    <widget class="QComboBox" name="formatCombo"/>
   </item>
   <item row="1" column="0" colspan="3">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Rows</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QRadioButton" name="allRowsRadio">
        <property name="text">
         <string>All (Full Fetch)</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QRadioButton" name="displayedRowsRadio">
        <property name="text">
         <string>Displayed Only</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QRadioButton" name="selectedRowsRadio">
        <property name="text">
         <string>Selected Only</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="4" column="0">
       <widget class="QCheckBox" name="includeRowHeaderCheck">
        <property name="text">
         <string>Include &amp;Row Numbers</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Columns</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QRadioButton" name="allColumnsRadio">
        <property name="text">
         <string>All in Row</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QRadioButton" name="selectedColumnsRadio">
        <property name="text">
         <string>Selected Only</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>182</width>
          <height>31</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="3" column="0">
       <widget class="QCheckBox" name="includeColumnHeaderCheck">
        <property name="text">
         <string>&amp;Include Column Headers</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QLabel" name="TextLabel1_2">
     <property name="text">
      <string>&amp;Separator (CSV):</string>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
     <property name="buddy">
      <cstring>separatorEdit</cstring>
     </property>
    </widget>
   </item>
   <item row="2" column="2" colspan="2">
    <widget class="QLineEdit" name="separatorEdit"/>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QLabel" name="TextLabel2">
     <property name="text">
      <string>&amp;Delimiter (CSV):</string>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
     <property name="buddy">
      <cstring>delimiterEdit</cstring>
     </property>
    </widget>
   </item>
   <item row="3" column="2" colspan="2">
    <widget class="QLineEdit" name="delimiterEdit"/>
   </item>
   <item row="4" column="3">
    <spacer name="Spacer2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="5" column="2" colspan="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>formatCombo</tabstop>
  <tabstop>separatorEdit</tabstop>
  <tabstop>delimiterEdit</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>formatCombo</sender>
   <signal>activated(int)</signal>
   <receiver>toResultListFormatUI</receiver>
   <slot>formatChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>134</x>
     <y>24</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>toResultListFormatUI</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>136</x>
     <y>176</y>
    </hint>
    <hint type="destinationlabel">
     <x>44</x>
     <y>123</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>toResultListFormatUI</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>235</x>
     <y>178</y>
    </hint>
    <hint type="destinationlabel">
     <x>62</x>
     <y>95</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
